infra/src/main/kotlin/fi/fta/geoviite/infra/ratko/RatkoPushDao.kt

```diff
@@ -205,7 +218,7 @@ class RatkoPushDao(jdbcTemplateParam: NamedParameterJdbcTemplate?) : DaoBase(jdb
                 assetId = trackNumberId
                     ?: locationTrackId
                     ?: switchId
-                    ?: error { "Encountered Ratko push error without asset! id: $errorId" },
```

Comment:
Olen korjaillut näitä siellä ja täällä: error-funktio ei ota lambdaa sisäänsä vaan vain Any olion jolle sanotaan toString -> tämän muotoiset virheet jättää viesteihin pelkkää puppua.